src/Behat/Config/Suite.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config;
+
+final class Suite implements SuiteConfigInterface
```

Comment:
Once we agree in #1547 whether or not to use interfaces for the DTOs and what to call them this might need to be updated.